Examples/Python/tests/test_examples.py

```diff
@@ -1021,5 +1021,7 @@ def test_full_chain_odd_example(tmp_path):
     assert script.exists()
     env = os.environ.copy()
     env["NEVENTS"] = "1"
-    env["ACTS_LOG_FAILURE_THRESHOLD"] = "WARNING"
+    # increased threshold because of vertexing
+    # see https://github.com/acts-project/acts/pull/1299
+    env["ACTS_LOG_FAILURE_THRESHOLD"] = "FATAL"
```

Comment:
Right.  This is where it gets tricky then. I guess under "normal" operation, there should be no ERRORs, or conversely, ERRORs (or even WARNINGs) should signal some kind of problem with the code.

Now that the vertexing throws ERRORs but they're not intrinsically fatal anymore, I'm not entirely sure what the best approach here is.

Reply:
I thought the situation is similar. we ultimately don't want to have errors under such nice conditions but for now we have some. I thought this is due to not having an ambiguity resolution? or do you think there is another problem upstream?

Reply:
I'd say the difference is this here right now isn't really a "bug" in the strict sense.

Reply:
true. I also don't like to push the threshold up again but if we want to add vertexing right now I don't see an alternative. or is there?

Reply:
We could make it configurable I guess...

Reply:
you mean not run the vertexing in this case but somewhere else for example?

Reply:
I was just thinking of making it configurable whether the vertexing algorithm logs this as `WARNING`/`ERROR` or just as `INFO`.